doc/hotspot-style.md

```diff
@@ -1070,6 +1070,8 @@ and other supported compilers may not have anything similar.
 * Unrestricted Unions
 ([n2544](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2008/n2544.pdf))
 
+* Covariant return types
```

Comment:
Covariant return types are a C++03 feature, so are covered by the blanket
permission for such here:
https://github.com/openjdk/jdk/blame/master/doc/hotspot-style.md#L385-L386

So the exclusion could be just deleted, rather than changing to explicit
permission.  Though an argument for explicit permission is to make the removal
of the long-standing exclusion more obvious.  I'm fine with either, but
thought it was worth mentioning.

Reply:
Thanks for the clarification. I removed the explicit permission.